### PR TITLE
@eessex => Modernize files in articles app and add /news route

### PR DESCRIPTION
**Pull request by kanaabe**
- Some coffeescript deprecation
- Add express handler for /news that goes to a temporary page under admin flag

Addresses: https://artsyproduct.atlassian.net/browse/GROWTH-378

TODO
- [x] Assess if we need `editorialForm` anymore in favor of `/signup/editorial` (we don't. previously we had to check if the user was signed up for editorial in the routes of /articles but we're showing the modal for logged out users now. only in mobile do we need the check in which case we handle separately)

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please assign someone to your PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: src/desktop/apps/articles/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+app.locals.toSentence = toSentence
+
+app.get('/posts', routes.redirectMagazine)
+app.get('/magazine', routes.redirectMagazine)
+app.get('/articles', routes.articles)
+app.get('/news', adminOnly, routes.news)
+app.get(sd.TEAM_BLOGS, routes.teamChannel)
+app.get('/venice-biennale-2015', routes.section)
+app.get('/gallery-insights/opt-in', (_, res) => {
+  res.redirect('/gallery-insights')
```
Comment (eessex) [inline src/desktop/apps/articles/index.js]: I'm not familiar with the `(_, res)` syntax, what is the underscore for?

File: src/desktop/apps/articles/test/routes.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      teamChannel(req, res, next)
+      Backbone.sync.args[0][2].success(channel)
+      Backbone.sync.args[1][2].data.ids.length.should.equal(4)
+      Backbone.sync.args[1][2].success(fixtures.article)
+      res.render.args[0][0].should.equal('team_channel')
+      res.render.args[0][1].channel.get('name').should.equal(channel.name)
+    })
+
+    it('nexts if channel is not a team channel', () => {
+      const channel = _.extend(_.cloneDeep(fixtures.channel), {
```
Comment (eessex) [inline src/desktop/apps/articles/test/routes.js]: What happens if it is a team channel and gets an error?  Maybe good to add a test for that outcome as well.

File: src/desktop/apps/articles/index.js (hunk 1 above)
Comment (kanaabe) [reply src/desktop/apps/articles/index.js]: I'm just using the underscore as a parameter name, I could have used `(req, res)` but the `_` designates that it isn't used.

Comment (eessex) [reply src/desktop/apps/articles/index.js]: Thanks, good to know!

**Pull request merged**